### PR TITLE
{mpi?}[GCC/4.9.2, gompi/2015a] GPI2 1.1.1, 1.2.0 (REVIEW)

### PR DESCRIPTION
**Pull request by tvandera**
GPI2 is a GASPI implementation. Build configs for 2 versions of GPI2 and 2 toolchains (one with and one without MPI). Also includes gawk for dependency.

### CODE REVIEW

File: easybuild/easyconfigs/g/GPI2/GPI2-1.2.0-gompi-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    'files': ['bin/gaspi_run'],
+    'dirs': []
+}
+
+buildopts = ' CPPFLAGS=-DGPI2_WITH_MPI && mv bin/gaspi_run.ssh bin/gaspi_run'
+
+files_to_copy = [
+                        (["bin/gaspi_run","bin/ssh.spawner","bin/gaspi_cleanup", "bin/gaspi_logger"], "bin"),
+                        "lib64", "tests", "include"
+                ]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GPI2/GPI2-1.2.0-gompi-2015a.eb]: please fix indent:

```python
files_to_copy = [
  (["bin/gaspi_run","bin/ssh.spawner","bin/gaspi_cleanup", "bin/gaspi_logger"], "bin"),
  "lib64", "tests", "include",
]
```

File: easybuild/easyconfigs/g/GPI2/GPI2-1.2.0-GCC-4.9.2.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    'files': ['bin/gaspi_run'],
+    'dirs': []
+}
+
+buildopts = ' && mv bin/gaspi_run.ssh bin/gaspi_run'
+
+files_to_copy = [
+                        (["bin/gaspi_run","bin/ssh.spawner","bin/gaspi_cleanup", "bin/gaspi_logger"], "bin"),
+                        "lib64", "tests", "include"
+                ]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GPI2/GPI2-1.2.0-GCC-4.9.2.eb]: please fix indent

File: easybuild/easyconfigs/g/GPI2/GPI2-1.1.1-gompi-2015a.eb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    'files': ['bin/gaspi_run'],
+    'dirs': []
+}
+
+buildopts = ' CPPFLAGS=-DGPI2_WITH_MPI && mv bin/gaspi_run.ssh bin/gaspi_run'
+
+files_to_copy = [
+                        (["bin/gaspi_run","bin/ssh.spawner","bin/gaspi_cleanup", "bin/gaspi_logger"], "bin"),
+                        "lib64", "tests", "include"
+                ]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GPI2/GPI2-1.1.1-gompi-2015a.eb]: please fix indent

File: easybuild/easyconfigs/g/GPI2/GPI2-1.1.1-GCC-4.9.2.eb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    'files': ['bin/gaspi_run'],
+    'dirs': []
+}
+
+buildopts = ' && mv bin/gaspi_run.ssh bin/gaspi_run'
+
+files_to_copy = [
+                        (["bin/gaspi_run","bin/ssh.spawner","bin/gaspi_cleanup", "bin/gaspi_logger"], "bin"),
+                        "lib64", "tests", "include"
+                ]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GPI2/GPI2-1.1.1-GCC-4.9.2.eb]: please fix indent

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/796e192d35271cfc0080 for a full test report.

**Comment by boegel** [comment]
@tvandera: good to go once the tiny style remarks are resolved

**Comment by tvandera** [comment]
Fixed indents. Hope this is sufficient. If not, let me know.

**Comment by boegel** [comment]
@wpoely86: good test case for your style checker? ;)

**Comment by migueldiascosta** [comment]
@tvandera, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**